doc/make.jl

```diff
@@ -1,152 +0,0 @@
-# Install dependencies needed to build the documentation.
```

Comment:
fortunately, it looks like AppVeyor won't give this PR a passing mark without this (but FreeBSD will). Need to merge a couple PRs to the doc packages first.